lib/ansible/module_utils/irmc.py

```diff
@@ -0,0 +1,143 @@
+
+# FUJITSU LIMITED
+# Copyright 2018 FUJITSU LIMITED
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division)
+__metaclass__ = type
+
+import traceback
+import json
+
+try:
+    import requests
```

Comment:
Can you use `ansible.module_utils.urls` instead of `requests`, which Ansible ships with.

Reply:
will do.